bunsen-avro/src/main/java/com/cerner/bunsen/avro/tools/GenerateSchemas.java

```diff
@@ -19,6 +20,8 @@
  */
 public class GenerateSchemas {
 
+  private static final String DELIMITER = ";";
```

Comment:
nit: you could make this as `public` and can use it in the [1] and other places in GenerateSchemasTest.java

[1] https://github.com/cerner/bunsen/pull/104/files#diff-d44e198d385c46339fe43cf3757e9dd8de8159080e1cfa61a845cb258836afbcR64

Reply:
Great suggestion. Done: 8841e32